tests/testthat/line_breaks_and_other/other-out.R

```diff
@@ -0,0 +1,30 @@
+# = replaced with <-
+a <- 3
+data_frame(a = 3)
+
+# semicolon
+a
+b
+c
+d
+
+
+# quotes
+"text with 'quotes'"
+'text with "quotes"'
+
+
+# linebreak after special
+a %>%
+  b() %>%
+  c()
+
+# adding brackets in pipes
+a %>%
+  b() %>%
+  c()
+
+# adding brackets in pipes combined with line breaks
+a %>%
```

Comment:
I think we should allow single-line pipes if all operations are on the same line.